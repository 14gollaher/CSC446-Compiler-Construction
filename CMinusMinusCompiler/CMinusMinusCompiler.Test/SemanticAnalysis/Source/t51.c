int a,b,c;
float x,y,z;

int main()
{
	return -1;
}
