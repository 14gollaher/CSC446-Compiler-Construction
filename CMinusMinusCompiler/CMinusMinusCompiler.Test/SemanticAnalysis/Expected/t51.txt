** Symbol Table at Depth 2 **
Lexeme          Class        Depth     Offset    Type         Size      Value(r)
--------------------------------------------------------------------------------


** Symbol Table at Depth 1 **
Lexeme          Class        Depth     Offset    Type         Size      Value(r)
--------------------------------------------------------------------------------
main            Function     1         -         IntToken     0         -
x               Variable     1         6         FloatToken   4         -
y               Variable     1         10        FloatToken   4         -
z               Variable     1         14        FloatToken   4         -
a               Variable     1         0         IntToken     2         -
b               Variable     1         2         IntToken     2         -
c               Variable     1         4         IntToken     2         -


Completed processing t51.c
